### PR TITLE
Update to 1.21.3

### LINKED ISSUE #55: 1.21.2 / 1.21.3 support? :D
title ^^

### PR DESCRIPTION
**Pull request by Onako2**
Update stuff to 1.21.3 and set environment to client (How to run it on the server?)

Current issues:
- Trinkets doesn't work because it didn't update yet

Closes #55

### CODE REVIEW

**Comment by legoraft** [comment]
> Update stuff to 1.21.3 and set environment to client (How to run it on the server?)

Thanks for setting environment to client by the way, I've seemed to have missed that.

File: src/client/java/com/armorhud/mixin/client/armorHudMixin.java [hunk 1]
```diff
@@ -85,7 +85,7 @@ private void renderArmorPiece(DrawContext context, float x, float y, RenderTickC
 
 		context.drawItem(player, stack, 0, 0, 1);
 
-		context.drawItemInSlot(this.client.textRenderer, stack, 0,0);
+		context.drawStackOverlay(this.client.textRenderer, stack, 0,0);
```
[the remark this answers is not in the document]
Comment (Onako2) [reply src/client/java/com/armorhud/mixin/client/armorHudMixin.java:88]: Yarn Mappings doing yarn things xD

**Comment by Onako2** [comment]
> You said that the runClient task doesn't work, does this also mean that the final jar doesn't work on the fabric loader?

It would work but because of the modImplementation stuff

**Comment by legoraft** [comment]
> It would work but because of the modImplementation stuff

So the mod just works, but the task itself doesn't work?

File: src/client/java/com/armorhud/mixin/client/armorHudMixin.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (legoraft) [reply src/client/java/com/armorhud/mixin/client/armorHudMixin.java:88]: Always changing stuff...

**Pull request merged**